run_desktop_tests.sh

```diff
@@ -31,7 +31,7 @@ cargo fmt --all -- --check
 cd ../..
 
 echo "Running Clippy lints..."
-cargo clippy --all-targets --features std --features with_nfc -- -A clippy::new_without_default -D warnings
+cargo clippy --all-targets --features std -- -A clippy::new_without_default -D warnings
```

Comment:
Very nice, that you got to work around the import problem! Can you also add an additional check with NFC enabled here and below?